proteus/NumericalSolution.py

```diff
@@ -7,6 +7,7 @@
 
 import os
 import numpy
+numpy.set_printoptions(threshold='nan')
```

Comment:
This was a statement used for debugging. Removed as of https://github.com/erdc-cm/proteus/pull/450/commits/15f7670c1764fdf15eac0f639a0e2d470fe6bc37